charts/studio/values.yaml

```diff
@@ -335,6 +335,34 @@ postgresql:
         # -- Postgres database
         database: iterativeai
 
+# -- Ray cluster settings group
+ray:
+  # -- Ray enabled
+  enabled: true
```

Comment:
Do we want to have it enabled by default?

Reply:
Probably not for people, but yes for our continuous integration tests. Maybe I can set it to `false` and then override it on the workflow?

Reply:
```suggestion
  enabled: false
```